packages/twenty-front/src/modules/views/view-filter-value/utils/resolveBooleanViewFilterValue.ts

```diff
@@ -1,7 +1,2 @@
-import { ViewFilter } from '@/views/types/ViewFilter';
-
-export const resolveBooleanViewFilterValue = (
-  viewFilter: Pick<ViewFilter, 'value'>,
-) => {
-  return viewFilter.value === 'true';
-};
+export const resolveBooleanViewFilterValue = (viewFilterValue: string) =>
```

Comment:
We can keep the previous function, we don't strive to compact the code in this codebase, it's okay to keep things explicit and less compact.